test/integration/sequelize.test.js

```diff
@@ -1132,8 +1134,17 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
   describe('import', function() {
     it('imports a dao definition from a file absolute path', function() {
       var Project = this.sequelize.import(__dirname + '/assets/project');
+      expect(Project).to.exist;
+    });
 
+    it('imports a dao definition from a file compiled with babel', function () {
+      var es6project = babel.transformFileSync(__dirname + '/assets/es6project.es6', {
+        presets: ['es2015']
+      }).code;
+      fs.writeFileSync(__dirname + '/assets/es6project.js', es6project);
+      var Project = this.sequelize.import(__dirname + '/assets/es6project');
       expect(Project).to.exist;
+      fs.unlink(__dirname + '/assets/es6project.js');
```

Comment:
This will be leftover on test failures i assume then? Maybe we should include a gitignore in assets for this file.